infer/models/c/src/libc_basic.c

```diff
@@ -682,6 +682,9 @@ unsigned sleep(unsigned seconds) {
 #ifdef __APPLE__ // define __WAIT_STATUS as int * on mac
 #define __WAIT_STATUS int*
 #endif
+#ifdef __GNUC__  // define __WAIT_STATUS as int * with glibc
```

Comment:
Ok so it looks like `__WAIT_STATUS` got simplified into `int *` in newest glibc-2.24:
https://www.sourceware.org/ml/libc-alpha/2016-08/msg00212.html

You can check it with `__GLIBC_PREREQ(2, 24)` Example usage:
http://www.boost.org/doc/libs/1_33_1/boost/config/platform/linux.hpp